frontend/src/common/config.ts

```diff
@@ -32,12 +38,17 @@ class ConfigWrapper {
 
   private constructor(data: ConfigDto) {
     this.data = data;
+    this.serverTimeDiff = Math.floor(Date.parse(data.serverTime).valueOf() / 1000) - Math.floor(Date.now() / 1000);
   }
 
   public get(): ConfigDto {
     return this.data;
   }
 
+  public getServerTimeDiff(): number {
+    return this.serverTimeDiff;
+  }
+
```

Comment:
see above

Reply:
Implemented in https://github.com/cryptomator/hub/pull/164/commits/96a104ec834fc75db9695a822462c5ca6c0e2f0f and https://github.com/cryptomator/hub/pull/164/commits/e373fb727a461e684eae89efcd068326e0d67611